### ISSUE TITLE
[NbConvertApp] ERROR | Error while converting '../tutorials/W1D3_ModelFitting/student/W1D3_Tutorial1.ipynb'

### ISSUE THREAD

**Issue by jerlich** [NeuromatchAcademy/course-content#262]
```jupyter
nbconvert.preprocessors.execute.CellExecutionError: An error occurred while executing the following cell:
------------------
#@title

#@markdown Execute this cell to visualize estimated models

fig, axes = plt.subplots(ncols=3, figsize=(18, 4))
for theta_hat, ax in zip(theta_hats, axes):

  # True data
  ax.scatter(x, y, label='Observed')  # our data scatter plot

  # Compute and plot predictions
  y_hat = theta_hat * x
  ax.plot(x, y_hat, color='r', label='Fit')  # our estimated model

  ax.set(
      title= fr'$\hat{{\theta}}$= {theta_hat}, MSE = {mse(x, y, theta_hat):.2f}',
      xlabel='x',
      ylabel='y'
  );

axes[0].legend()
------------------

---------------------------------------------------------------------------
NameError                                 Traceback (most recent call last)
<ipython-input-7-b60d2a652106> in <module>
      4
      5 fig, axes = plt.subplots(ncols=3, figsize=(18, 4))
----> 6 for theta_hat, ax in zip(theta_hats, axes):
      7
      8   # True data

NameError: name 'theta_hats' is not defined
```

**Comment by ebatty** [comment]
theta_hats is defined in the previous cell so not sure what's happening there...

**Comment by ebatty** [comment]
ah perhaps it's because it is defined only in the solution and commented out in the exercise cell so stripping solutions creates a problem

**Comment by ebatty** [comment]
I have made a PR to fix this by uncommenting in the exercise cell

**Comment by ebatty** [comment]
I'm checking for this issue in other W1D3 tutorial notebooks now

**Comment by mwaskom** [comment]
@jerlich the student versions are by definition not executable (because they are incomplete). They should be written such that the only error you would see is `NotImplementedError` (and it looks like @ebatty has a fix in for that here), but in general we can't guarantee that you'll be able to execute them.

**Comment by mwaskom** [comment]
Closing as I think we've resolved this on Slack — @jerlich  ping if you disagree.

**Comment by jerlich** [comment]
But... would be better to get the not implemented exception than NameError

**Comment by mwaskom** [comment]
@ebatty fixed that in #264
